Sources/_StringProcessing/Regex/Match.swift

```diff
@@ -123,7 +123,7 @@ extension Regex {
   /// Find the first match in a substring
   ///
   /// Returns `nil` if no match is found and throws on abort
-  public func firstMatch(_ s: Substring) throws -> Regex<Output>.Match? {
+  public func firstMatch(in s: Substring) throws -> Regex<Output>.Match? {
```

Comment:
I thought `Output` was renamed to `RegexOutput`?

Reply:
Regex's generic parameter is still named `Output`.